create-electron-documentation/readme.md

```diff
@@ -0,0 +1,35 @@
+# `create-electron-documentation`
+
+The goal of this package is to help developers create the scaffolding
+for [Electron] examples faster.
+
+## How to use it?
+
+Electron's documentation is under the `/docs` folder of the [main repo].
+You will have to clone it locally and call the following script from one
+of the folders under `/docs` (e.g.: [`/docs/tutorial`][tutorials]):
+
+```console
+npm create electron-documentation
+```
+
+The script will ask you only 2 things that cannot be empty strings:
```

Comment:
This section might be better with a literal example output of the CLI.

Something like:

````markdown
The script will prompt you to enter the documentation page title and description, for example:
```sh
> npm create electron-documentation
> Title:
> my-new-doc-page
> Description:
> A description for my new doc page
```
````